sparse_strips/vello_hybrid/src/scene.rs

```diff
@@ -134,32 +120,46 @@ impl Scene {
         if !self.paint_visible {
             return;
         }
-        flatten::fill(
-            self.level,
-            path,
-            self.transform,
-            &mut self.line_buf,
-            &mut self.flatten_ctx,
-        );
+
         let paint = self.encode_current_paint();
-        self.render_path(self.fill_rule, paint);
+        self.fill_path_with(path, self.transform, self.fill_rule, paint, self.anti_alias);
+    }
+
+    fn fill_path_with(
```

Comment:
[nit]: It might be helpful to add short comments for `fill_path_with` and `stroke_path_with`.